### ISSUE TITLE
(aws-cdk): IAM Diff CLI error due to updates in `tables` dependency

### ISSUE THREAD

**Issue by mnlkrs** [aws/aws-cdk#12306]
The [table](https://github.com/gajus/table) dependency [recently updated](https://github.com/gajus/table/commit/645060e0e99061e8de29bb0a263f689fb9ada069) their validation to not allow additional properties. The CloudFormation diff however, seems to be using an unknown property `useWordWrap`.

This results in the CLI terminating prematurely and printing an error in the table dependency.

### Reproduction Steps

1. Setup a project with IAM changes
2. Run `cdk diff`

### What did you expect to happen?

A diff should be output.

### What actually happened?

```bash
❯ yarn cdk diff
yarn run v1.22.5
$ cdk diff
Stack ProjectStack
IAM Statement Changes
config {
  border: {
    topBody: '\u001b[90m─\u001b[39m',
    topJoin: '\u001b[90m┬\u001b[39m',
    topLeft: '\u001b[90m┌\u001b[39m',
    topRight: '\u001b[90m┐\u001b[39m',
    bottomBody: '\u001b[90m─\u001b[39m',
    bottomJoin: '\u001b[90m┴\u001b[39m',
    bottomLeft: '\u001b[90m└\u001b[39m',
    bottomRight: '\u001b[90m┘\u001b[39m',
    bodyLeft: '\u001b[90m│\u001b[39m',
    bodyRight: '\u001b[90m│\u001b[39m',
    bodyJoin: '\u001b[90m│\u001b[39m',
    joinBody: '\u001b[90m─\u001b[39m',
    joinLeft: '\u001b[90m├\u001b[39m',
    joinRight: '\u001b[90m┤\u001b[39m',
    joinJoin: '\u001b[90m┼\u001b[39m'
  },
  columns: {
    '0': { width: 1, useWordWrap: true },
    '1': { width: 13, useWordWrap: true },
    '2': { width: 6, useWordWrap: true },
    '3': { width: 13, useWordWrap: true },
    '4': { width: 13, useWordWrap: true },
    '5': { width: 14, useWordWrap: true }
  },
  drawHorizontalLine: [Function: drawHorizontalLine]
}
errors [
  {
    dataPath: '/columns/0',
    message: 'should NOT have additional properties',
    params: { additionalProperty: 'useWordWrap' },
    schemaPath: '#/additionalProperties'
  },
  {
    dataPath: '/columns/1',
    message: 'should NOT have additional properties',
    params: { additionalProperty: 'useWordWrap' },
    schemaPath: '#/additionalProperties'
  },
  {
    dataPath: '/columns/2',
    message: 'should NOT have additional properties',
    params: { additionalProperty: 'useWordWrap' },
    schemaPath: '#/additionalProperties'
  },
  {
    dataPath: '/columns/3',
    message: 'should NOT have additional properties',
    params: { additionalProperty: 'useWordWrap' },
    schemaPath: '#/additionalProperties'
  },
  {
    dataPath: '/columns/4',
    message: 'should NOT have additional properties',
    params: { additionalProperty: 'useWordWrap' },
    schemaPath: '#/additionalProperties'
  },
  {
    dataPath: '/columns/5',
    message: 'should NOT have additional properties',
    params: { additionalProperty: 'useWordWrap' },
    schemaPath: '#/additionalProperties'
  }
]
Invalid config.
```

### Environment

  - **CDK CLI Version  :** v1.81.0
  - **Framework Version:** v1.81.0
  - **Node.js Version:** v12.4.0
  - **OS               :** MacOS Big Sur 11.0.1
  - **Language (Version):** TypeScript 4.1.3

### Other

--- 

This is :bug: Bug Report

**Comment by c22** [comment]
Note to maintainers, this issue can block deployments as `cdk deploy` will run a diff first.

Workaround for blocked deployments is to rollback to previous version or add `--require-approval never`.

**Comment by mzazon** [comment]
Ran into this today after 1.81.0 -> 1.82.0 on our codebase and @c22 's suggestion worked great as a workaround.

**Comment by majdarbash** [comment]
Seems there's an upgrade to "**table**" package which is breaking the diff command.
Try to add the following to package.json:
`
"resolutions": {
    "table": "6.0.4"
  },
`
Worked for me (running cdk 1.78.0 and 1.82.0). 
It is a temporary fix until the issue is solved.

**Comment by dyoshikawa** [comment]
I encountered a similar problem on v1.82.0.

**Comment by mnlkrs** [comment]
This is fixed in #12330, however it's now released yet.
